cmd/kubeadm/app/phases/token/bootstrap_test.go

```diff
@@ -104,13 +120,22 @@ func TestCreateBootstrapConfigMapIfNotExists(t *testing.T) {
 		},
 	}
 
-	file, err := ioutil.TempFile("", "")
-	if err != nil {
-		t.Fatalf("could not create tempfile: %v", err)
-	}
-	defer os.Remove(file.Name())
+	tempFiles := make([]*os.File, 2)
+	for i, _ := range tempFiles {
+		file, err := ioutil.TempFile("", "")
+		if err != nil {
+			t.Fatalf("could not create tempfile: %v", err)
+		}
+		defer os.Remove(file.Name())
 
-	file.Write([]byte(testConfig))
+		config := testConfig
+		if i == 1 {
```

Comment:
I think it's much better to create a slice of strings and loop through that instead, and from there populate `files`